receiver/hostmetricsreceiver/internal/metadata/generated_metrics.go

```diff
@@ -59,6 +59,8 @@ type metricStruct struct {
 	ProcessDiskIo               MetricIntf
```

Comment:
This will need to be regenerated after the suggestions are merged to change from `IntSum` to `Sum`